data/service/api/v1/devicetokens.go

```diff
@@ -0,0 +1,45 @@
+package v1
+
+import (
+	data "github.com/tidepool-org/platform/data/service"
+	"github.com/tidepool-org/platform/devicetokens"
+	"github.com/tidepool-org/platform/request"
+	platform "github.com/tidepool-org/platform/service"
+	"github.com/tidepool-org/platform/service/api"
```

Comment:
TL;DR If some way of consistently distinguishing package names is needed, I feel like that should be moved into the package name itself. Then everyone's tooling inserts the proper name automatically, and no one has to guess how the name is elongated. 

----

Lots of rambling thoughts below... You've been warned.

I'm not against consistency, but having to guess at names isn't consistent. My editor (and I suspect yours too?) inserts package names using their full names automatically. I didn't choose these names myself. They are not abbreviated. 

Part of what I don't like about the suggestion to extend the names, is that they're inconsistent with the [recommendations of the Go authors](https://go.dev/doc/effective_go#package-names). It forces me to guess at the various names for the packages ([is it serviceAPI, or serviceApi](https://go.dev/wiki/CodeReviewComments#initialisms)?). 

I don't like having to guess, and we'll have fewer nits moving forward, because no matter how hard I try, I'm not going to remember to rename package references that where ***automatically added*** by my editor without me even having to do anything.

Here are some concrete options of alternative ways to solve this without needing to remember how to "extend" names:

### Option 0 (What we have now)

> package "service" 

with

> "github.com/tidepool-org/platform/data/service"

### Option 1

> package "dataservice" 

with

> "github.com/tidepool-org/platform/dataservice"

(Note the lack of slash between the last occurrence of "data" and "service".)

### Option 2

If you feel like anything related to the data service absolutely must live under /data:

> package "dataservice" 

with 

> "github.com/tidepool.org/platform/data/dataservice"

### Option 3

We could likewise do:

> package "dataservice" 

with

> "github.com/tidepool.org/platform/data/service"

But that seems to needlessly flaunt the convention of a package's name matching the basename of the directory in which its found.

Reply:
Actually, the original intention was:

**Option 4**

> package "dataService"

with

> "github.com/tidepool-org/platform/data/service"

where the capitalization in the import was an indicator of a package name separator.

However, I suppose it really doesn't matter, so we can continue to each do what seems normal to us and just deal with the differences between authors.